### PR TITLE
simple todo list with click and completed show

### PR DESCRIPTION
**Pull request by PuniaMohit**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+import Tasks from './todo.json'
+import { useState } from "react"
 
 function App() {
-  return (
-    <></>
+
+  const [indexCheck, setIndexCheck] = useState([])
+
+  function change(event, index) {
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit what do you mean by function name change? also why are you not using es6 functional conventions

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,37 @@
 import './App.css';
+import Tasks from './todo.json'
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit todo.json exporting Tasks? why this much issue in naming conventions?

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,12 @@
...
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+.container{
```
Comment (meanstack9716) [inline src/App.css]: @PuniaMohit code is not formatted

File: src/App.js [hunk 4]
```diff
@@ -1,8 +1,37 @@
...
 
 function App() {
-  return (
-    <></>
+
+  const [indexCheck, setIndexCheck] = useState([])
+
+  function change(event, index) {
+    const { checked } = event.target
+  if(checked===true){
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit why checking checked === true? is if(checked) not working?

File: src/App.js [hunk 5]
```diff
@@ -1,8 +1,37 @@
...
+    setIndexCheck(oldArray => [...oldArray, index])
+  }
+  else{
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit code is not formatted

File: src/App.js [hunk 6]
```diff
@@ -1,8 +1,37 @@
...
+    setIndexCheck(indexCheck.filter(item => item !== index))
+  }
+  }
+
+  return (<div className='container'>
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit code is not formatted

File: src/App.js [hunk 7]
```diff
@@ -1,8 +1,37 @@
...
+    <h3>Todo App</h3>
+      {Tasks.map((element, index) => {
```
Comment (meanstack9716) [inline src/App.js]: Tasks variable names should not start from Capital case @PuniaMohit

File: src/App.js [hunk 8]
```diff
@@ -1,8 +1,37 @@
...
+        return <>
+          <div style={{ display: 'flex', justifyContent: 'space-between' }} key={index}>
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit do not use inline css

File: src/App.js [hunk 9]
```diff
@@ -1,8 +1,37 @@
...
+            <div>
+              <input type="checkbox" id={index} name={index} value={element.task} onClick={(event) => { change(event, index) }} key={index}/>
+              <label for={index}>{element.task}{index}</label>
+            </div>
+            <div>{ 
+            indexCheck.includes(index) ? <button variant="outline-secondary">Complete</button> : <></>}</div>
```
Comment (meanstack9716) [inline src/App.js]: @PuniaMohit code format issue

File: src/App.js [hunk 10]
```diff
@@ -1,8 +1,37 @@
...
+          </div>
+          <hr />
```
Comment (meanstack9716) [inline src/App.js]: why use hr? @PuniaMohit  can you do it why border styling

File: src/todo.json [hunk 11]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "task":"abc"
```
Comment (meanstack9716) [inline src/todo.json:3]: @PuniaMohit json is not formatted

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 4 above)
Comment (PuniaMohit) [reply src/App.js]: sir by this condition, I am checking index number of that tasks is in array or not. I made an array which consits index number of all selected tasks.

**Pull request merged**